### PR TITLE
complete testing about_blocks.rb

### PR DESCRIPTION
**Pull request by trungle1612**

### CODE REVIEW

File: about_blocks.rb [hunk 1]
```diff
@@ -54,31 +54,31 @@ def yield_tester
...
+    assert_equal 11, add_one[10]
+  end #OK
 
   def test_stand_alone_blocks_can_be_passed_to_methods_expecting_blocks
     make_upper = lambda { |n| n.upcase }
     result = method_with_block_arguments(&make_upper)
-    assert_equal __, result
-  end
+    assert_equal "JIM", result
+  end # Ok
```
Comment (tangkiemandao) [inline about_blocks.rb]: I don't know why use "JIM" at here.

File: about_blocks.rb [hunk 2]
```diff
@@ -54,31 +54,31 @@ def yield_tester
...
-  end
+    assert_equal 11, add_one[10]
+  end #OK
 
   def test_stand_alone_blocks_can_be_passed_to_methods_expecting_blocks
     make_upper = lambda { |n| n.upcase }
     result = method_with_block_arguments(&make_upper)
-    assert_equal __, result
-  end
+    assert_equal "JIM", result
```
Comment (trungle1612) [inline about_blocks.rb:80]: I don't know why use "JIM" in this line.

File: about_blocks.rb [hunk 3]
```diff
@@ -40,8 +40,8 @@ def many_yields
   def test_methods_can_call_yield_many_times
     result = []
     many_yields { |item| result << item }
-    assert_equal __, result
-  end
+    assert_equal [:peanut, :butter, :and, :jelly], result
```
Comment (trungle1612) [inline about_blocks.rb:43]: You should use %i[peanut, butter, and, jelly] instead of "[:peanut, :butter, :and, :jelly]" :D.

**Review by trungle1612 — COMMENTED** [review]
fixed

**Pull request merged**